addons/l10n_eu_service/models/res_config_settings.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    l10n_eu_services_eu_country = fields.Boolean('Is European country?', compute='_compute_l10n_eu_services_european_country')
+
+    @api.depends('company_id')
+    def _compute_l10n_eu_services_european_country(self):
+        self.l10n_eu_services_eu_country = bool(self.env.ref('base.europe').id in self.company_id.country_id.country_group_ids.ids)
```

Comment:
More logical to use 'country in group' then country group in country groups?  (logical code vs debatable perf gains)

Reply:
@jco-odoo agreed! And, I've rearranged commits as you suggested and fixed those typos. So, I think today is the day ;)